profiles/micro-profile/pom.xml

```diff
@@ -17,7 +17,7 @@
         implementations of the profile/specification.
     </description>
 
-    <artifactId>kumuluzee-micro-profile</artifactId>
+    <artifactId>kumuluzee-microProfile</artifactId>
```

Comment:
Is this intended change?

Reply:
Yes, the naming of the MP artifacts was inconsistent. But this is however breaking change so discussion is welcome.

Reply:
We have to preserve backward compatibility. Either leave kumuluzee-micro-profile or define both (kumuluzee-micro-profile and kumuluzee-microProfile).